content/filterx/filterx-parsing/cef/cef-parser-options/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Options of CEF parsers"
+weight:  100
+---
+<!-- This file is under the copyright of Axoflow, and licensed under Apache License 2.0, except for using the Axoflow and AxoSyslog trademarks. -->
+
+The `parse_cef` FilterX function has the following options.
+
+## pair_separator
+
+Specifies the character or string that separates the CEF fields from each other. Default value: `|` .
```

Comment:
Specifies the character or string that separates the key-value pairs in the extensions. Default value: ` ` (space).